desc/transform.py

```diff
@@ -377,6 +380,12 @@ def build(self):
                     self.grid.nodes, d, unique=True
                 )
 
+        if self.method == "jitable":
```

Comment:
what is this doing that makes it jitable? precomputing everything?

Reply:
its the `unique=False` part that makes it jitable. Possibly more expensive since theres some duplicate work if the are duplicate nodes, but this is usually for unstructured grids so it should be fine.